notebooks/Adapter_With_Embeddings.ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
Can we add a title and a short introduction explaining the goal of this notebook?

Reply:
That is a very good point! I added title and introduction to the notebook